frontend/src/services/api.ts

```diff
@@ -61,6 +63,52 @@ export let getAdequacies = memoize(
   (params: PostAdequaciesRequest) => `${params.method}-${params.providers.join(',')}-${params.service_area_ids.join(',')}`
 )
 
+export async function getStaticRPs(selectedDataset: Dataset | null){
+    const res = await fetch(getStaticRPUrl(selectedDataset))
+    const body = await res.json()
+    return <StaticRepresentativePointsResponse> body
+}
+
+export async function getStaticAdequacies(selectedDataset: Dataset | null, method: Method){
+    const res = await fetch(getStaticAdequacyUrl(selectedDataset, method))
+    const body = await res.json()
+    return <PostAdequaciesResponse> body
+}
+
+export async function getStaticDemographics(selectedDataset: Dataset | null){
+    const res = await fetch(getStaticDemographicsUrl(selectedDataset))
+    const body = await res.json()
+    return <PostCensusDataResponse> body
+}
+
+function getStaticRPUrl(selectedDataset: Dataset | null): string {
+    const rootUrl = CONFIG.staticAssets.representativePoints.rootUrl
+    if (!selectedDataset){
```

Comment:
why throw this error instead of just having it be a required param?

Reply:
The type of `selectedDataset` in the store is defined as `Dataset` or `null`, which as best as I can tell necessitates nullchecking it everywhere :/